### PR TITLE
[WIP] Add bookinfo traffic shifting tests

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
Added tests to check traffic shifting from v1 to v2 by 90/10 and 80/20 %.

### CODE REVIEW

File: samples/bookinfo/istio.io_tutorial/route-rule-reviews-80-20.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2017,2018 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (gyliu513) [inline samples/bookinfo/istio.io_tutorial/route-rule-reviews-80-20.yaml]: no need to add the copyright for yaml template

Comment (vadimeisenbergibm) [reply samples/bookinfo/istio.io_tutorial/route-rule-reviews-80-20.yaml]: What is the rationale though? The deployment YAML files have copyright, e.g. https://github.com/istio/istio/blob/master/samples/bookinfo/kube/bookinfo.yaml

**Comment by vadimeisenbergibm** [comment]
Closing, waiting for https://github.com/istio/istio/pull/4185

**Pull request closed without merging**